.github/workflows/karpenter-comment.yaml

```diff
@@ -0,0 +1,22 @@
+name: Karpentercomment
+on:
+  issue_comment:
+    types: [created, edited, deleted]
+
+jobs:
+  approval-comment:
+    if: github.repository == 'aws/karpenter' && contains(github.event.comment.html_url, '/pull/') && contains(github.event.comment.body, '/karpenter snapshot')
```

Comment:
WDYT?
```suggestion
    if: contains(github.event.comment.html_url, '/pull/') && contains(github.event.comment.body, '/karpenter-snapshot')
```

Reply:
`/karpenter` in future can become bunch of different commands that all start with `/karpenter` I want the following strings to be parameters being passed to it